dts/bindings/mtd/nxp,s32-qspi-device.yaml

```diff
@@ -20,3 +20,15 @@ properties:
       Memory alignment in bytes, used to calculate padding when performing
       unaligned accesses.
       If not provided, 1 byte alignment will be selected.
+
+  program-page-size:
```

Comment:
It's look to me this properties should be marked as `required: true`. Pls explain why the default value is chosen

Reply:
`It's look to me this properties should be marked as required: true`: if this is correct, pls update.
`Pls explain why the default value is chosen`, i meant add explanation in devicetree bindings, not via PR's comment

Reply:
If a default is set, the description text needs to explain where that default comes from

Reply:
I update explanation for  the default value